kubernetes/helm/pinot/values.yaml

```diff
@@ -130,6 +133,10 @@ broker:
   updateStrategy:
     type: RollingUpdate
 
+  envFrom: []
```

Comment:
Can you add the comments for example value and the reference doc link you put in the PR here. Same for all other places.

Reply:
Thanks @fx19880617, I added examples, a brief description and a ref link for each.